### PR TITLE
Fix scheduling lists

### PR DESCRIPTION
**Pull request by rosendi**

### CODE REVIEW

File: src/blocks/modal/scheduleModal/scheduleModal.ejs [hunk 1]
```diff
@@ -1,48 +1,50 @@
...
-					}
-				}) %>
-			</div>
-		</div>
-	</div>
+    <div class="<%- self.generateClassName('modal', self.modifiers) %>">
+        <div class="modal__title"><%- i18n('Запланируйте выпуск рассылки') %></div>
+
+        <div class="modal__content">
+            <%= this.include(DatePicker, {
```
Comment (zouxuoz) [inline src/blocks/modal/scheduleModal/scheduleModal.ejs]: use `self`

File: src/blocks/mailing/issueMailing/issueMailing.js [hunk 2]
```diff
@@ -78,6 +80,26 @@ define(function(require) {
...
+                models: self.models,
+
+                confirm: function(rawDate) {
+                    var departAt = moment(rawDate, 'DD.MM.YYYY HH:mm').format('YYYY-MM-DD HH:mm');
+
+                    self.setLoading(event.currentTarget);
+
+                    self.draft.reschedule(departAt, { silent: true }).then(function() {
+                        self.render();
+                        self.showNotification('Время отправки выпуска изменено');
```
Comment (zouxuoz) [inline src/blocks/mailing/issueMailing/issueMailing.js]: use `i18n`

File: src/blocks/table/mailings/mailings.ejs [hunk 3]
```diff
@@ -46,7 +46,7 @@
 
                     <tbody>
                         <% if (Page.current.params.type == 'delayed') { %>
-                            <% if (table.hasDelayedMailings()) { %>
+                            <% if (table.list.length > 0) { %>
```
Comment (zouxuoz) [inline src/blocks/table/mailings/mailings.ejs]: duplicate check from line 11

File: src/blocks/modal/scheduleModal/scheduleModal.ejs (hunk 1 above)
Comment (rosendi) [inline src/blocks/modal/scheduleModal/scheduleModal.ejs]: done

File: src/blocks/mailing/issueMailing/issueMailing.js (hunk 2 above)
Comment (rosendi) [inline src/blocks/mailing/issueMailing/issueMailing.js]: done

File: src/blocks/table/mailings/mailings.ejs (hunk 3 above)
Comment (rosendi) [inline src/blocks/table/mailings/mailings.ejs]: done

**Comment by zouxuoz** [comment]
lgtm

**Pull request merged**